CONTRIBUTING.md

```diff
@@ -50,7 +52,7 @@ Go can be [installed](https://golang.org/doc/install#install) from the official
 
 [Snap](https://snapcraft.io/go) may also be used to install Go on Linux.
 
-    snap install go --channel=1.18/stable --classic
+    snap install go --channel=latest/stable --classic
```

Comment:
not latest, best to stick to a version, eg 1.19

Reply:
I have latest installed lmao